ib_cni_test.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+export GOROOT=${GOROOT:-/usr/local/go}
+export GOPATH=${WORKSPACE}
+export PATH=/usr/local/go/bin/:$GOPATH/src/k8s.io/kubernetes/third_party/etcd:$PATH
+export TIMEOUT=${TIMEOUT:-300}
+
+export POLL_INTERVAL=${POLL_INTERVAL:-10}
+export NETWORK=${NETWORK:-'192.168'}
```

Comment:
what is "192.168" is that a network?

Reply:
yes. I need to see that it on the interface inside.